src/content/docs/infrastructure/host-integrations/host-integrations-list/redis/sonarqube-integration.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Sonarqube web app monitoring integration
+tags:
+    - New Relic integrations
+    - SonarQube
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Flutter web application.
+redirects: 
+
+---
+
+import browserFlutterWebDashboard from 'images/browser_screenshot-full_flutter-web-dashboard.png'
+
+Our New Relic Infrastructure agent and New Relic Flex to the instrument and monitor your SonarQube platform. Our SonarQube integration gives you a pre-built dashboard that allows you to evaluate the data from your project, such as line duplications, flaws, and other mistakes in order to diagnose and optimize the code.
+
+<img
+    title="New Relic Flutter web app dashboard"
+    alt="New Relic Flutter web app dashboard"
+    src={browserFlutterWebDashboard}
+/>
+<figcaption>
+After setting up our Flutter integration, we give you a dashboard for your Flutter web app metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1 (optional): Install the infrastructure agent [#infra-install]
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running. 
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent) for the host containing your Flutter app.
+
+### Step 2: Configure New Relic Flex [#browser-install] 
+
+Our SonarQube integration relies on our New Relic Flex tool to work. To learn more and check requirements review our [Flex docs](docs/infrastructure/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/)
+
+1. Generate a SonarQube user token at [**User > My Account > Security**](https://docs.sonarqube.org/9.8/user-guide/user-account/generating-and-using-tokens/#using-a-token).
+2. Name your token.
+3. Make sure that you've selected **User token** as your **Type**.
+4. Create a nri-flex configuration file like `nri-flex-config.yml` using this path: `etc/newrelic-infra/integrations.d/ `
+4. Add these configuration lines in the config file:
+```
+integrations:
+  - name: nri-flex
+    # interval: 30s
+    config:
+      name: sonarFlex
+      apis:
+        - name: sonarSampleStatus
+          commands:
+            - run: curl -u <sonarqube_user_token>: http://localhost:9000/api/monitoring/metrics
+              split_by: ": "
+          snake_to_camel: true
+```
+
+8. Restart your application. 
+
+Wait five minutes and then proceed to finding your data in New Relic. 
+
+## Find your data [#find-data]
+
+To get your SonarQube dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
```

Comment:
For some reason the marketplace link is directing me to a signup page. Worth looking into.